app/controllers/suggestion_controller.rb

```diff
@@ -0,0 +1,75 @@
+# Route for release2
+get "/" do
+  erb :index
+end
+
+# Route for release1
+# get "/" do
+#   redirect "/suggestions"
+# end
+
+
+get "/suggestions" do
+  @suggestions = Suggestion.all
+  erb :"/suggestions/index"
+end
+
+get "/suggestions/:id" do
+  @suggestion = Suggestion.find_by(id: params[:id])
+  puts @suggestion
+  if @suggestion
+    erb :"/suggestions/show"
+  else
+    "Suggestion not found"
+  end
+end
+
+post "/suggestions" do
+  @suggestion = Suggestion.new(title: params[:suggestion][:title],
+    description: params[:suggestion][:description], user_id: session[:user_id])
+  puts params
+  if @suggestion.save
+    redirect "/suggestions"
+  else
+    "Cannot add the suggestion"
+  end
+end
+
+post "/suggestions/login" do
+  @suggestion = Suggestion.new(title: params[:suggestion][:title],
+    description: params[:suggestion][:description], user_id: session[:user_id])
+  if @suggestion.save
```

Comment:
Why is the route on this action `/suggestion/login`? (Looks like this action creates a new suggestion.)

Reply:
Yes, should be in suggestions/new. 
I'll see how to modify this, and keep the same UX.